README.rst

```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
     cd astroid
     python3 -m pip install -e .
 
+Share your code style
+-------
+
+Place this badge at the top of your repository to let others know your project uses pylint.
+
+    .. image:: https://img.shields.io/badge/linting-pylint-green
```

Comment:
The badge I made is just a proposition, if you have something better in mind it's possible to change the url to get another badge automatically.

Reply:
[![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=ef8336)](https://pycqa.github.io/isort/)

I really liked the isort github badge. So maybe something like a mix of pylint colours - Blue and yellow would be great like below one:

![linting: pylint](https://img.shields.io/badge/%20linting-pylint-%231674b1?style=flat&labelColor=ddff00)](https://github.com/PyCQA/pylint)

Do share your thoughts @Pierre-Sassoulas ?

Reply:
>  maybe something like a mix of pylint logo colours

Great idea, but the blue and yellow of the pylint logo are the same than python, we might as well use something else as it's not distinguishing at all in the python ecosystem.

I like  https://img.shields.io/badge/linting-pylint-yellowgreen, it's less flashy than green or yellow and look reasonably good in light/dark theme.

Reply:
Let's go with yellowgreenthen 👍